dockers/chart-releaser/main.go

```diff
@@ -146,6 +149,17 @@ func main() {
 	log.Printf("Successfully stored all packages and index file")
 }
 
+func downloadDependencies(ctx context.Context, chartPath string) error {
```

Comment:
This can be resolved by adding `--dependency-update` flag to the `package` command.

Reply:
I had no idea! Thanks, let me use it.